bin/compile

```diff
@@ -68,10 +68,10 @@ set-env PATH '/app/.indyno/vendor/redis/bin:$PATH'
 PASSWORD=`openssl rand -hex 16`
 
 # placeholder username h was added for compatibility with old Redis clients (https://devcenter.heroku.com/changelog-items/1932) and is incompatible with Redis 6 AUTH
-if [[ "${VERSION}" > 5.* ]]; then
-	export REDIS_URL="redis://:$PASSWORD@localhost:6379/"
-else
+if [[ "${VERSION}" < 6 ]]; then
```

Comment:
Small nit (also from the original) - `<` is used for _text_ comparisons in shell, rather than _numeric_ comparisons. This should be something like:

```bash
if [ $(echo "${VERSION} < 6" | bc -l) == 0 ]; then
```

Reply:
This gets trickier because `${VERSION}` in this context won't be a number, but a `x.y.z` string. However, I can apply this on the first number from the version, which will be more accurate than the current text comparison. I'm going to try this, as the simpler text comparison works but makes it much easier to make mistakes.

Thank you!

Reply:
Ah, OK. In that case, if `dpkg` is available at compile time, then `dpkg --compare-versions "${VERSION}" "lt" "6"` would be a valid replacement and handles all the dot-separated format complexity.

Reply:
I like this, thank you! I see this is what the PHP buildpack does in a few places and the version comparison is much more clear.